src/chromo/models/dpmjetIII.py

```diff
@@ -146,6 +140,21 @@ def _cross_section(self, kin=None, precision=None):
 
         # TODO set more cross-sections
 
+    def set_hA_AA_glauber_trials(self, ntrials=1000):
```

Comment:
A pure setter should not have a default argument. This should be a property
```
@property
glauber_trials(self):
     """<Documentation goes here, default values of 1000 is only mentioned in the docs>"""
     return self._lib.dtglgp.jstatb

@glauber_trials.setter
def _(self, ntrials):
      <content of set_hA_AA_glauber_trials goes here>
```

Reply:
Nope. The default number should be set by DPMJET’s initialization and mentioned only in the docs if at all.

Reply:
@afedynitch It sounds like we agree, right?

Reply:
I changed the code to use properties.

Reply:
I still have some comments on the name and the doc string which I added below.